src/i18n/pt-br/ui.ts

```diff
@@ -103,6 +104,9 @@ export default UIDictionary({
 	'feedback.categoryGroupLabel': 'Escolha a categoria do feedback',
 	'feedback.issue': 'Problema',
 	'feedback.createIssue': 'Criar um problema no GitHub',
+	'feedback.createIssue.description': 'A maneira mais rápida de alertar nosso time sobre um problema.',
+	'feedback.sendFeedback': 'Enviar feedback',
+	'feedback.sendFeedback.description': 'Nos envia um feedback diretamente.',
```

Comment:
```suggestion
	'feedback.sendFeedback.description': 'Nos envie um feedback diretamente.',
```

I think `envie` would make more sense in here.

Reply:
I think this is refurring to what the button does, not the user.

At least that is what I got from the other descriptions.

Reply:
That makes sense. LGTM, then.